ngrinder-controller/src/main/java/org/ngrinder/script/repository/FileEntryRepository.java

```diff
@@ -229,7 +221,6 @@ public FileEntry findOne(User user, String path, SVNRevision revision) {
 			if (nodeKind == SVNNodeKind.NONE) {
 				return null;
 			}
-			outputStream = new ByteArrayOutputStream();
 			SVNProperties fileProperty = new SVNProperties();
 			// Get File.
 			repo.getFile(path, revision.getNumber(), fileProperty, outputStream);
```

Comment:
make ```try``` start here

Reply:
Above try block enclose SVNException, If move the block to here, we should have nested try-catch.